src/main/java/racingCar/domain/car/RacingCars.java

```diff
@@ -0,0 +1,28 @@
+package racingCar.domain.car;
+
+import racingCar.domain.moveStrategy.MoveBehavior;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+    private final List<RacingCar> racingCarList;
+
+    public RacingCars(final String[] carNames) {
+        this.racingCarList = initRacingCars(carNames);
+    }
+
+    private List<RacingCar> initRacingCars(String[] carNames) {
+        return Arrays.stream(carNames).map(RacingCar::new).collect(Collectors.toList());
+    }
+
+    public List<RacingCar> move(MoveBehavior moveBehavior) {
+        racingCarList.forEach(racingCar -> racingCar.move(moveBehavior.isMove()));
+        return racingCarList;
+    }
+
+    public List<RacingCar> getRacingCarList() {
+        return racingCarList;
```

Comment:
List<RacingCar>의 참조를 직접 외부에 노출하게 되면 final로 선언되어 있더라도 `List<RacingCar>` 리스트 내부의 값은 변경할 수 있습니다.

아래와 같이 `unmodifiableList` 같은 메소드를 활용해보세요

```suggestion
        return Collections.unmodifiableList(racingCarList);;
```